setup-README.rst

```diff
@@ -0,0 +1,187 @@
+Google Cloud Python Client
```

Comment:
Let's just make this *really* small and only mention the deprecation. Also go ahead and move setup.py/etc into legacy/google-cloud

Reply:
lmk if it is small enough now. I left some other things. I can drop absolutely everything also but I figured the license text and build state should still be present. Happy to remove more.